src/finiteVolume/finiteVolumeSolver.cpp

```diff
@@ -252,37 +248,18 @@ void ablate::finiteVolume::FiniteVolumeSolver::RegisterRHSFunction(RHSArbitraryF
 
 void ablate::finiteVolume::FiniteVolumeSolver::RegisterPreRHSFunction(PreRHSFunctionDefinition function, void* context) { preRhsFunctions.emplace_back(function, context); }
 
-void ablate::finiteVolume::FiniteVolumeSolver::EnforceTimeStep(TS ts, ablate::solver::Solver& solver) {
```

Comment:
So each solver gets a ComputeTimeStep function instead of the option that existed before?

Reply:
Yes, every solver get the option to implement a physics based timeStep